### PR TITLE
Delete photos

### PR DESCRIPTION
**Pull request by andrewxhill**
fixes https://github.com/textileio/textile-mobile/issues/346

### CODE REVIEW

File: App/Config/ReduxPersist.ts [hunk 1]
```diff
@@ -90,13 +90,26 @@ const migrations: MigrationManifest = {
...
+  },
+  7: (persistedState) => {
+    const state = persistedState as any
+    return {
+      ...state,
+      preferences: {
+        ...state.preferences,
+        tourScreens: {
+          ...state.preferences.tourScreens,
+          threads: true
```
Comment (andrewxhill) [inline App/Config/ReduxPersist.ts]: Converted the threads onboarding screen into the same as the wallet one. now that threads are the default view, it was creating a flash of onboarding every time it loaded with the old way anyway

File: App/SB/views/PhotoDetail/index.js [hunk 2]
```diff
@@ -194,19 +218,21 @@ const mapStateToProps = (state, ownProps) => {
...
   const path = thread.name === 'default' ? '/photo' : '/thumb'
+  const source = photo ? {url: 'file://' + photo.id + '.png'} : {url: 'file://.png'}
+
   return {
     photo,
-    date: photo.date.split('T')[0],
-    key: photo.id + path,
-    source: {url: 'file://' + photo.id + '.png'}, // <-- in case RN uses to know things
+    date: photo && photo.date ? photo.date.split('T')[0] : '',
+    key: photo && photo.id ? photo.id + path : path,
```
Comment (andrewxhill) [inline App/SB/views/PhotoDetail/index.js]: all this is just making it safe the if the view doesn't unmount in time before the image remove request is done that it wont error out

**Comment by andrewxhill** [comment]
okay, updated android bridge. gtg

File: App/Redux/PreferencesRedux.ts [hunk 3]
```diff
@@ -45,7 +45,8 @@ export const initialState: PreferencesState = {
   onboarded: false,
   verboseUi: false,
   tourScreens: {
-    wallet: true
+    wallet: true,
+    threads: true
```
Comment (asutula) [inline App/Redux/PreferencesRedux.ts]: I don't understand why these would initially be `true`. Seems that the tour UI would never show for new users then. Maybe I'm missing something.

Comment (andrewxhill) [reply App/Redux/PreferencesRedux.ts]: true == yes, show it

**Pull request merged**